src/main/java/com/gooddata/md/visualization/Bucket.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+package com.gooddata.md.visualization;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.io.Serializable;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Bucket implements Serializable {
+
+    private static final long serialVersionUID = -7718720886547680021L;
+    private String localIdentifier;
```

Comment:
are these field intended to be mutable? If yes , add setters, of no make them final. (In general we prfer mutable DTOs if the client should have ability to PUT them back)